algorithms/gr-bfsrh/python2/bfs.py

```diff
@@ -0,0 +1,42 @@
+from collections import defaultdict
+
+class Graph:
+  def __init__(self):
+    self.graph = defaultdict(list)
+
+  def addEdge(self,u,v):
+    self.graph[u].append(v)
+
+
+  def BFS(self, s):
+
+    visited = [False] * (len(self.graph))
+
+    queue = []
+
+    queue.append(s)
+    visited[s] = True
+
+    while queue:
+      s = queue.pop(0)
+      print (s, end = " ")
```

Comment:
This is a python3 supported print statement. You need to use it in a python2 supported way.